daemon/daemon/loadbalancer.go

```diff
@@ -95,20 +94,16 @@ func (d *Daemon) svcAdd(feL3n4Addr types.L3n4AddrID, beL3n4Addr []types.L3n4Addr
 	}
 
 	svc := types.LBSVC{
-		FE:  feL3n4Addr,
-		BES: beL3n4AddrCpy,
+		FE:     feL3n4Addr,
+		BES:    beL3n4AddrCpy,
+		Sha256: feL3n4Addr.L3n4Addr.SHA256Sum(),
```

Comment:
I really dislike all cap names. They are completely unreadable to me. Let's do proper camel case.